Source/EasyNetQ.Tests.Messages/project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "2.0.2-netcore0001",
```

Comment:
It would be a good idea to fashion the description, authors, tags, etc similar to this: https://github.com/EasyNetQ/EasyNetQ/blob/feature/netcore/Source/EasyNetQ/project.json